src/main/kotlin/fi/metatavu/vp/deliveryinfo/thermometers/Thermometer.kt

```diff
@@ -0,0 +1,59 @@
+package fi.metatavu.vp.deliveryinfo.thermometers
+
+import fi.metatavu.vp.deliveryinfo.persistence.Metadata
+import fi.metatavu.vp.deliveryinfo.sites.Site
+import jakarta.persistence.*
+import java.time.OffsetDateTime
+import java.util.*
+
+/**
+ * Entity for thermometers
+ */
+@Entity
+@Table(name = "thermometer")
+class Thermometer {
+    @Id
+    lateinit var id: UUID
+
+    @Column(nullable = false)
+    lateinit var hardwareSensorId: String
+
+    @Column(nullable = false)
+    lateinit var espMacAddress: String
+
+    @Column
+    var name: String? = null
+
+    @ManyToOne
+    var site: Site? = null
```

Comment:
```suggestion
    lateinit var site: Site
```
According to the changelog it is non-nullable